apps/astarte_pairing/lib/astarte_pairing/fdo/rendezvous/core.ex

```diff
@@ -24,11 +24,12 @@ defmodule Astarte.Pairing.FDO.Rendezvous.Core do
   @es256 -7
   @es256_identifier 1
   @cose_sign1_tag 18
+  @http 3
+  @load_balancer_port 4003
 
-  def get_rv_to2_addr_entries(first_entry, first_entry_ip, second_entry, second_entry_ip) do
-    rv_entry1 = CBORCore.build_rv_to2_addr_entry(first_entry_ip, first_entry, 8080, 3)
-    rv_entry2 = CBORCore.build_rv_to2_addr_entry(second_entry_ip, second_entry, 8080, 3)
-    {:ok, [rv_entry1, rv_entry2]}
+  def get_rv_to2_addr_entry(full_dns_address, pre_existing_entries \\ []) do
```

Comment:
Nit: there's no such thing as a "DNS address"
```suggestion
  def get_rv_to2_addr_entry(full_dns_name, pre_existing_entries \\ []) do
```